src/WebJobs.Extensions.DurableTask.Analyzers/WebJobs.Extensions.DurableTask.Analyzers.csproj

```diff
@@ -8,7 +8,7 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.DurableTask.Analyzers</PackageId>
-    <PackageVersion>0.3.1</PackageVersion>
+    <PackageVersion>0.3.2</PackageVersion>
```

Comment:
Is this supposed to be 0.3.1 instead of 0.3.2? Just thought the analyzer was going from 0.3.0 to 0.3.1 but I may be wrong.

Reply:
0.3.1 was already released, so we need to do another analyzer release to fix this.